src/shared/Dropdown.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, JSX, Show, createSignal } from "solid-js";
+
+import { ChevronDown } from "lucide-solid";
+import Button from "./Button";
+
+
+const Dropdown: Component<{ children: JSX.Element; label: string }> = (
+  props
+) => {
+  const [open, setOpen] = createSignal(false);
+
+  return (
+    <div class="inline-block">
+      <Button
+        onClick={() => {
+          setOpen(!open());
+        }}
+      >
+        <ChevronDown />
+        {props.label}
+      </Button>
+      <Show when={open()}>
+        <div class="dropdown-menu">
+          {props.children}
+        </div>
+      </Show>
```

Comment:
At the moment since this is a simple div, it shifts the layout around when the dropdown is opened. Would you be willing to give [floating-ui](https://floating-ui.com/) a shot so the dropdown menu can float above the other content when it's open?

Reply:
solved with latest commit, used [popper](https://github.com/lxsmnsyc/solid-popper) bc floating ui was being cringe